ui/decredmaterial/editor.go

```diff
@@ -173,27 +149,56 @@ func (e Editor) Layout(gtx layout.Context) layout.Dimensions {
 	})
 }
 
-func (e Editor) editorLine(gtx C) D {
+func (e Editor) editorLayout(gtx C) D {
+	if e.Bordered {
+		return e.editorRectangle(gtx, func(gtx C) D {
+			return e.editorSection(gtx)
+		})
+	}
+
+	return e.editorSection(gtx)
+}
+
+func (e Editor) editorSection(gtx layout.Context) layout.Dimensions {
```

Comment:
the entire code results in the editor widget. Having a method named "editorSection" is a little confusing.

Reply:
Since i will be wrapping the editor in the border widget, the editorSection function is necessary. I will rename this to editor for simplicity.